### PR TITLE
Level Database setup

### PR DESCRIPTION
**Pull request by Yarin18**

### CODE REVIEW

File: shared/src/androidMain/kotlin/di/PlatformModules.android.kt [hunk 1]
```diff
@@ -7,24 +7,31 @@ import db.database.car.CarDatabase
...
+import db.database.level.LevelDatabaseWrapper
 import db.database.user.UserDatabase
 import db.database.user.UserDatabaseWrapper
 import org.koin.dsl.module
 
 actual fun platformModules() = module {
 
     single {
-        val driver = AndroidSqliteDriver(ChargeHubDb.Schema, get(), "chargehubdb.db")
+        val driver = AndroidSqliteDriver(ChargeHubDb.Schema, get(), DATABASE_NAME)
```
Comment (DanneD-Wisemen) [inline shared/src/androidMain/kotlin/di/PlatformModules.android.kt:22]: I saw you changed this to extract the driver, don't forget to implement this everywhere

Comment (DanneD-Wisemen) [reply shared/src/androidMain/kotlin/di/PlatformModules.android.kt:22]: + nice job extracting the db name

File: shared/src/commonMain/kotlin/di/Helper.kt [hunk 2]
```diff
@@ -3,6 +3,8 @@ package di
 import org.koin.core.context.startKoin
 import org.koin.dsl.KoinAppDeclaration
 
+const val DATABASE_NAME : String = "chargehubdb.db"
```
Comment (DanneD-Wisemen) [inline shared/src/commonMain/kotlin/di/Helper.kt]: I would create a new file in the commonMain/kotlin called Constants.kt

**Review by DanneD-Wisemen — CHANGES_REQUESTED** [review]

File: shared/src/androidMain/kotlin/di/PlatformModules.android.kt (hunk 1 above)
Comment (Yarin18) [reply shared/src/androidMain/kotlin/di/PlatformModules.android.kt:22]: Extracting the driver is only possible for iOS as far as Ive been able to figure out, and thank you

**Pull request merged**